templates/web/oxfordshire/reports/_other_districts-list.html

```diff
@@ -67,12 +67,12 @@ <h2>View reports by district</h2>
   <p class="[% show_district_hidden_class %]"><a href="?show_districts=1" class="js-ward-select-multiple">Select multiple districts?</a></p>
   <form method="get" action="">
     <section class="full-width">
-      <ul class="item-list item-list--wards">
+      <ul class="item-list item-list--wards govuk-checkboxes govuk-checkboxes--small">
         [% FOR child IN parish_ward.values.sort('name') ~%]
         [% IF child.type == 'DIS' %]
-        <li class="item-list__item item-list--wards__item">
```

Comment:
`item-list--wards__item` has been removed from the HTML, but is still present in a number of cobrand CSS (only ever copying what reports__item is doing, looks like), so they can all be removed/checked?